win32ss/gdi/eng/surface.c

```diff
@@ -127,6 +127,10 @@ SURFACE_AllocSurface(
     PSURFACE psurf;
     SURFOBJ *pso;
     PVOID pvSection;
+    BOOL bExtraBitsNeeded = FALSE;
+
+    DPRINT("cy '%ld' iFormat '%ld' cjWidth '%ld' cjBufSize '%ld' pvBits '%p'.\n",
+        cy, iFormat, cjWidth, cjBufSize, pvBits ? pvBits : 0);
```

Comment:
A question, no change-request yet:
that " ? pvBits : 0" part is really necessary for the %p-format-string to handle 0 properly?

Reply:
It shouldn't. It would matter only if it was for displaying a string, in which case your format string would have `%s` and the pointer thing would be something like: `pStr ? pStr : ""`  (in order to avoid getting the standard `(null)` replacement string or a crash in case the printf formatter doesn't do the null check).

Reply:
Removed extra unneeded code. Thanks.